src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useNavigate } from 'react-router-dom';
+
+import Header from 'components/Header';
+import { buttonVariant } from 'components/ui/Button';
+import PageHeading from 'components/ui/PageHeading';
+import PageText from 'components/ui/PageText';
+
+import styles from './NotFoundPage.module.css';
+
+const NotFoundPage = () => {
+  const navigate = useNavigate();
+
+  const handleGoBack = () => {
+    navigate(-1);
+  };
+
+  return (
+    <>
+      <Header />
```

Comment:
Header mustn't be here

Reply:
It should be there as there is no layout